catalog/spatial/registry/registry-app/pom.xml

```diff
@@ -75,6 +75,10 @@
             <groupId>org.codice.ddf.registry</groupId>
             <artifactId>registry-report-action-provider</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Okay looks like this isn't set up in <dependencyManagement> so you would need to explicitly add the <version> here